lib/bamboo.js

```diff
@@ -470,6 +470,53 @@ module.exports = function(host, username, password) {
             }
         )
     }
+    
+    /**
+     * creates a plan branch in Bamboo 
+     * 
+     * this call seems to be not documented, so we relay on some forum data:
+     * https://answers.atlassian.com/questions/215164/bamboo-rest-api-to-create-branches
+     * 
+     * note that for this to work, a repository needs to be link to Bamboo plan, since its not well documented
+     * I dont know how to handle multiple repositories  
+     * 
+     * On success - callback(null, true)
+     * On failure - callback(error)
+     *
+     * @param   {String} planKey - "PROJECT_KEY-PLAN_KEY"
+     * @param   {String} bambooBranchName - the name that will be displayed in Bamboo for the branch
+     * @param   {String} vcsBranchName - the branch name in the repository, this parameter depends on the version control used,
+     * in git its something like "refs/heads/BRANCH_NAME" - you can find it by going to .git/refs/heads folder in your file system 
+     * @param   {Object} params
```

Comment:
unspecified param, please remove this line, since your method do not accept param `params`.